bgpd/bgp_evpn.c

```diff
@@ -2282,13 +2285,14 @@ static int install_uninstall_route_in_vrfs(struct bgp *bgp_def, afi_t afi,
 	struct bgp *bgp_vrf;
 	struct listnode *node, *nnode;
 
-	/* Only type-2 routes go into a VRF */
-	if (!(evp->prefix.route_type == BGP_EVPN_MAC_IP_ROUTE))
+	/* Only type-2/type-5 routes go into a VRF */
```

Comment:
by the way, RT1 handling is not supported yet.

Reply:
yes, RT-1 is not yet supported. Processing for type-1 route not reach this point, it will dropped earlier. We will have to change this handling when we start supporting RT-1.